styles.css

```diff
@@ -1 +1,71 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.container {
+    background-color: #F7D3CF;
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50 px 50px;
+    grid-template-areas:
+            "header"
+            "textarea"
+            "aside-big"
+            "aside-small"
+            "aside-dynamic"
+            "empty"
```

Comment:
Jouw oplossing met "empty" lijkt perfect te werken, maar een leeg gridvakje mag eigenlijk alleen met een punt (.)  worden aangeduid. Dan kan dat vak dynamisch groter of kleiner worden (zie ook:  https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas#Leaving_a_grid_cell_empty)

Reply:
Bedankt voor jouw opmerkingen Martijn! Leuk om zo van elkaar de verschillende inzichten te zien en te leren.